src/CheckoutSdk/Financial/ActionBreakdown.cs

```diff
@@ -0,0 +1,42 @@
+using Checkout.Common;
+using System;
+
+namespace Checkout.Financial
+{
+    public class ActionBreakdown
+    {
+        public string BreakdownType { get; set; }
+
+        public float FxRateApplied { get; set; }

```

Comment:
Please migrate float to double, and also include `?`, all primitives, enums, dates must have the null marker